noxfile.py

```diff
@@ -136,6 +136,36 @@ def docs(session: Session) -> None:
     )
 
 
+@nox.session(python=python, venv_backend="conda")
+def schema_diagram(session: Session) -> None:
+    session.install("-e", "./jobmon_server")
+    outpath = Path(__file__).parent / "docsource" / "developers_guide" / "diagrams" / "erd.svg"
+    with tempfile.TemporaryDirectory() as tmpdir:
+        session.chdir(tmpdir)
+        session.run(
+            "jobmon_server", "init_db",
+            env={"JOBMON__DB__SQLALCHEMY_DATABASE_URI": "sqlite:///jobmon.db"}
+        )
+        session.run("docker", "pull", "schemacrawler/schemacrawler", external=True)
```

Comment:
Sounds like this session will only work on docker-enabled hosts. I'm not sure if the github actions server supports docker (for free), but probably not a huge problem